website/docs/cloud-docs/users-teams-organizations/2fa.mdx

```diff
@@ -13,7 +13,7 @@ User accounts can be additionally protected with two-factor authentication (2FA)
 
 ## Setting up Two-factor Authentication
 
-To reach your user security settings page, click the user icon in the upper right corner and choose "User Settings" from the menu.
+To reach your user security settings page, click the user icon in the upper right corner and choose "Account Settings" from the menu.
```

Comment:
This is a pro-active suggestion outside of your change, but we use bold with UI elements in the docs, so appreciate if we tweak this to:
```suggestion
To reach your user security settings page, click the user icon in the upper right corner and choose **Account Settings** from the menu.
```